docs/cloud/satellites/byoc/byoc.md

```diff
@@ -37,19 +37,19 @@ BYOC satellites are [available with an Enterprise plan](https://earthly.dev/pric
 
 ### Step 1: Configure Your Cloud Provider
 
-Follow the instructions on the [AWS CloudFormation](https://docs.earthly.dev/earthly-cloud/satellites/byoc/cloud-providers/aws) page to provision the required AWS resources. Right now, BYOC Satellites are only supported in AWS.
+There are multiple methods you can use to configure your cloud provider. Choose the one that works best for your organization:
 
+* [AWS CloudFormation](./aws/cloudformation.md)
+* [Terraform](./aws/terraform.md)
+* [Manual](./aws/manual.md)
 
-### Step 2: Install In Earthly
-
-After installing the required resources within your cloud provider; you must complete the installation by telling Earthly about your new configuration. Run:
+Follow the instructions for your selected installation method to provision the required resources. Right now, BYOC Satellites are only supported in AWS.
 
-```shell
-earthly cloud install <name>
-```
+### Step 2: Install In Earthly
 
-Where `<name>` is the name of the Installation, as specified by your cloud provider. Assuming it reports the status as `Green`, you should be good to go!
+After installing the required resources within your cloud provider; you must complete the installation by telling Earthly about your new configuration. Run the command in the documentation for your selected installation method to proceed.
 
+Assuming the installation reports the status as `Green`, you should be good to go!
```

Comment:
This step seems empty now (since its instructions are in the AWS sub-pages). Should we just remove it?

Reply:
I'm not sure how to remove it without feeling lost. We could maybe mention some stuff in step 1 about how it will take you through installation, and then I can push down the stuff about `Green` status. Will try real quick.

Reply:
I removed it. Let me know if this makes more sense.